.github/ISSUE_TEMPLATE/user-facing-feature.md

```diff
@@ -1,40 +1,59 @@
 ---
 name: User facing feature
-about: For internal use
-title: ''
-labels: ''
+about: For internal use only; Open a user-facing feature tracking file
+title: "[FEATURE] - "
+labels: ["type: internal-feature", "user type: internal"]
 assignees: ''
 
 ---
 
-## What - The user facing feature being implemented
+## What
+<!-- What the end user will see -->
 
 ## Why
+<!-- The reason why this user-facing feature is being implemented -->
 
-## Personas - Who will this affect?
+## Personas
+<!-- Who will this affect? -->
 
-- [ ] SPO
-- [ ] Dapp Devs
+- [ ] SPOs
+- [ ] dApp Devs
 - [ ] Exchanges
+- [ ] Wallets
+- [ ] 3rd party tools
+- [ ] ADA holders 
 
-## Acceptance Criteria
-
-## Definition of done
+## Definition of Done (DoD)
 
-- [ ] Code review
+- [ ] [Acceptance Criteria + User Stories & DoD](https://input-output.atlassian.net/wiki/spaces/QA/pages/3584229468/Acceptance+Criteria+DoD) created and singed-off (by PO, dev & test owners)
```

Comment:
Is it possible to have public facing documentations to set expectations external to IO as well?

Reply:
> Is it possible to have public facing documentations to set expectations external to IO as well?

100% - the usage examples/responses for the new user-facing functionalities should be in the PR descriptions that are already open to anybody.